lfs.c

```diff
@@ -5,7 +5,7 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 #include "lfs.h"
-#include "lfs_util.h"
+#include <lfs_util.h>
```

Comment:
Do these need to be angle-brackets?

Reply:
The angle brackets allow the user to add a custom lfs_util.h by giving their version precedence in the include order. If they don't have a custom lfs_util.h it will still use the one in this repo. I found this slightly useful but I can definitely workaround reverting.

Reply:
Oh interesting, is this a case that LFS_CONFIG does not work for?
https://github.com/littlefs-project/littlefs/blob/4c9146ea539f72749d6cc3ea076372a81b12cb11/lfs_util.h#L10-L20

It's a minor concern, but it is possible that the angle brackets could cause the wrong file to get included for users.